docs/releases/v1.6.0.md

```diff
@@ -27,19 +28,21 @@ No further action. Just apply this new kustomize project.
 
 ```bash
 $ kustomize build katalog/nginx | kubectl apply -f -
+# omitted output
 $ kustomize build katalog/dual-nginx | kubectl apply -f -
+# omitted output
 ```
 
 ### cert-manager
 
 This procedure was designed to apply the update at once. Jetstack team have written the upgrade path for every
 cert-manager minor release:
 
-- https://cert-manager.io/docs/installation/upgrading/upgrading-0.9-0.10/
-- https://cert-manager.io/docs/installation/upgrading/upgrading-0.10-0.11/
-- https://cert-manager.io/docs/installation/upgrading/upgrading-0.11-0.12/
-- https://cert-manager.io/docs/installation/upgrading/upgrading-0.12-0.13/
-- https://cert-manager.io/docs/installation/upgrading/upgrading-0.13-0.14/
+- [https://cert-manager.io/docs/installation/upgrading/upgrading-0.9-0.10/](https://cert-manager.io/docs/installation/upgrading/upgrading-0.9-0.10/)
```

Comment:
TIL: links are not automatically linked in github...
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links